custom-metrics-stackdriver-adapter/README.md

```diff
@@ -1,8 +1,8 @@
 # Custom Metrics - Stackdriver Adapter
 
 Custom Metrics - Stackdriver Adapter is an implementation of [Custom Metrics
```

Comment:
I'd rewrite this whole paragraph as:

The purpose of Custom Metrics - Stackdriver Adapter is to enable pod autoscaling based on Stackdriver custom metrics. It does it by implementing [Custom Metrics API] and [External Metrics API] using Stackdriver as a backend.

Reply:
Reworded a little bit, but I prefer the current order of these sentences. The most important part is, it's CM and EM API implementation for SD. The purpose is just a further explanation, it may also expand in the future. For instance, it's already more convenient for me to play with EM API than to query Stackdriver directly with API explorer :)